config/test.json

```diff
@@ -37,5 +37,14 @@
   },
   "livesInGcp": [
     "gcp"
-  ]
+  ],
+  "s3": {
```

Comment:
Is this added due to some thing otherwise crashing in lu-common? Feels off. A change in lu-common would be preferred IMHO.

Reply:
When removing s3 and ses configs from `test.json`, we get the following error:

![image](https://github.com/BonnierNews/lu-broker/assets/44803347/a1ee8dc8-b78d-43a2-9e21-e9e03558a556)

`s3.js` and `ses.js` assumes we have configs set up when we import lu-common. We could fix it, but I was worried it might be a bit out of scope for now since I've only got this week left before my holiday :)

Reply:
We probably can fix it this week though, if we prefer it that way

Reply:
It just feels odd adding something we're moving away from to a repository that doesn't even use it. It's not the most important thing to fix though I agree with that.